### PR TITLE
Remove No Bridge Fees annoucement banner

### PR DESCRIPTION
**Pull request by evandrosaturnino**
## Description 

- [X] - [Remove no bridge fees banner](https://github.com/threshold-network/token-dashboard/commit/413140adda129061de935652bfb4a0b81f6bea84)

## Notice 

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change? 

## Pull Request Type 

- [x] New Feature (Breaking/Non-breaking Change) 

## Issue (if applicable)

Closes issue #737

### CODE REVIEW

File: src/pages/Overview/index.tsx [hunk 1]
```diff
@@ -14,7 +12,6 @@ const Overview: PageComponent = () => {
     <Container maxW={{ base: "2xl", xl: "6xl" }} my={16}>
       <Image src={thresholdWordMark} mb={4} />
       <H1 mb={12}>Overview</H1>
-      {featureFlags.TBTC_V2 && <NoBridgeFeesBanner mb={"4"} />}
```
Comment (michalsmiarowski) [inline src/pages/Overview/index.tsx:17]: It looks like it only remove `NoBridgeFees` banner from overview page, but doesn't remove the badge from tbtc subpage:

image

Please take a look at #702 and #711. Ideally most of the changes applied there (or even all of them, not sure) should be reversed.

Comment (michalsmiarowski) [inline src/pages/Overview/index.tsx:17]: I'm just wondering - do we want to remove the banner completely? Or revert to the old banner which was "The new tBTC app is here"?

image

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left some changes to look at before the merge

File: src/pages/Overview/index.tsx (hunk 1 above)
Comment (evandrosaturnino) [reply src/pages/Overview/index.tsx:17]: Thanks for bringing that up, addressed in https://github.com/threshold-network/token-dashboard/pull/738/commits/947d8818d745429bb6d4a029e37981cf774e9b78

Comment (evandrosaturnino) [reply src/pages/Overview/index.tsx:17]: Good point, would like to know your opinions @erdogan and @SorinQ

Comment (erdogan) [reply src/pages/Overview/index.tsx:17]: Let's reinstate the banner, with a copy revision:  
- Replace "The NEW tBTC dApp is here!" with "Get your Bitcoin to work!" 
- Replace button label "Start Minting" with "Mint tBTC"

Comment (evandrosaturnino) [reply src/pages/Overview/index.tsx:17]: Thanks, addressed in [Add mint tbtc banner back](https://github.com/threshold-network/token-dashboard/pull/738/commits/059e0c979e19938e52d81d6b60fa5a3e1329c7b1)

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🔥

**Pull request merged**